src/storage/store/engine_tests.rs

```diff
@@ -878,9 +880,11 @@ mod tests {
         assert_eq!(stored_onchain_events.len(), 1);
 
         // Hub events are generated
-        let events = HubEvent::get_events(engine.db.clone(), 0, None, None).unwrap();
+        let mut events = HubEvent::get_events(engine.db.clone(), 0, None, None).unwrap();
         assert_eq!(1, events.events.len());
-        let received_event = event_rx.recv().await.unwrap();
+        let mut received_event = event_rx.recv().await.unwrap();
+        received_event.produced_at = 0;
+        events.events.get_mut(0).unwrap().produced_at = 0;
```

Comment:
Why do we need to set to 0?

Reply:
The assert compares the events with each other exactly. We use current timestamp for populating shard chunk timestamp so this timestamp field differs. 

It's kind of annoying to thread through an optional or default timestamp-- we have to modify the commit_shard_chunk function which is used a lot.

Reply:
added a comment -- the timestamp is not stored in the db but it is populated in the generated event based on the timestamp in the shard chunk. that time is now current time.